assets/css/translation.css

```diff
@@ -126,3 +126,116 @@ html[dir='rtl'] #srcLanguages {
     padding: 0 15px;
     position: relative;
 }
+
+#apertiumLogo {
+    margin-bottom: -.25em;
+}
+
+#applyingWidth {
+    width: 35%;
+}
+
+#applyingPadding {
+    padding: 5px 5px 0 0;
+}
+
+#translateDoc {
+    margin-top: 1em;
+}
+
+#fileDropBackdrop {
+    z-index: 1000;
+}
+
+#dropDocumentDiv {
+    left: 35%;
+    top: 35%;
+}
+
+#fileDropMask {
+    opacity: 0;
+    z-index: 10000;
+}
+
+#uploadErrorDiv {
+    margin-top: 1em;
+    min-height: 50px;
+}
+
+.progress {
+    margin-bottom: 0;
+}
+
+#fileUploadProgress {
+    width: 0%;
+}
+
+#fileLoading {
+    left: 50%;
+    top: 50%;
+}
+
+#morphAnalyzerOutput {
+    border-spacing: 0;
+}
+
+#selectLanguage {
+    padding: 5px 0 0 5px;
+}
+
+#mistakeId {
+    cursor: pointer;
+}
+
+#noticeMistake {
+    margin-top: 10px;
+}
+
+#maintainerAbout {
+    padding-bottom: 2em;
+}
+
+#gsocRef,
+#codeInRef {
+    text-decoration: none;
+}
+
+#gsocLogo {
```

Comment:
These logo rules don't really have anything to do with `translation` so they shouldn't be in this file. Maybe in general.css?